File: platforms/web/examples/core/mesh/Mesh.js
import { {Mesh, Vertex, Position, VertexValue} } from "fragmentcolor";

const mesh = new Mesh();
mesh.addVertex(Vertex.fromPosition(Position.Pos3([0.0, 0.5, 0.0])));
mesh.addVertex(Vertex.fromPosition(Position.Pos3([-0.5, -0.5, 0.0])));
mesh.addVertex(Vertex.fromPosition(Position.Pos3([0.5, -0.5, 0.0])));

File: platforms/python/examples/core/mesh/Mesh.py
from fragmentcolor import {Mesh, Vertex, Position, VertexValue}

mesh = Mesh()
mesh.add_vertex(Vertex.from_position(Position.Pos3([0.0, 0.5, 0.0])))
mesh.add_vertex(Vertex.from_position(Position.Pos3([-0.5, -0.5, 0.0])))
mesh.add_vertex(Vertex.from_position(Position.Pos3([0.5, -0.5, 0.0])))